src/corespotlight.cs

```diff
@@ -74,6 +74,10 @@ interface CSSearchableIndex {
 		[Export ("initWithName:protectionClass:")]
 		NativeHandle Constructor (string name, [NullAllowed] NSString protectionClass);
 
+		[iOS (9, 0), Mac (10, 11), MacCatalyst (13, 1), NoTV, NoWatch]
+		[Export ("initWithName:protectionClass:bundleIdentifier:options:")]
+		NativeHandle Constructor (string name, [NullAllowed] string protectionClass, string bundleIdentifier, nint options);
```

Comment:
Looks like `protectionClass` can be a strongly typed enum:

https://developer.apple.com/documentation/corespotlight/cssearchableindex/4288306-initwithname?changes=latest_minor&language=objc

Reply:
Agreed, let's add the enum